types/d3-delaunay/index.d.ts

```diff
@@ -178,46 +183,6 @@ export namespace Delaunay {
      */
     type GetCoordinate<P, PS> = (point: P, i: number, points: PS) => number;
 
-    /**
```

Comment:
See the "hull" property. It changed from this Node interface to an int32Array, so this interface is no longer needed.